### PR TITLE
Stackhpc/prod openstack pike magnum queens

### PR DESCRIPTION
**Pull request by brtkwr**
Add public key for  `$USER` into `authorized_keys` for `stack` user on all hosts:
```
10.41.253.[100: seed | 101: prod controller | 102: prod monitor | 103: alt-1 controller]
```

Prepare environment:
```
source venv/kayobe/bin/activate
source ../kayobe-config/kayobe-env
cd src/kayobe
export KAYOBE_VAULT_PASSWORD=****secret****
```

To build and deploy new magnum image:
```
kayobe overcloud container image build magnum --push -e kolla_source_version=stable/queens -e kolla_openstack_release=6.0.0.0 -e kolla_source_url=https://git.openstack.org/openstack/kolla
kayobe overcloud container image pull --kolla-tags magnum
kayobe overcloud service upgrade --kolla-tags magnum 
```

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -33,11 +33,11 @@
 #kolla_ansible_source_path:
 
 # URL of Kolla Ansible source code repository if type is 'source'.
-#kolla_ansible_source_url:
+kolla_ansible_source_url: https://www.github.com/stackhpc/kolla-ansible
```
Comment (dougszumski) [inline etc/kayobe/kolla.yml:36]: I think these are defaults in kayobe, so need to set them here:

kayobe]$ grep -r kolla_ansible_source_url
ansible/group_vars/all/kolla:kolla_ansible_source_url: "https://github.com/stackhpc/kolla-ansible"

File: etc/kayobe/kolla.yml [hunk 2]
```diff
@@ -112,13 +112,17 @@ kolla_build_blocks:
...
-    # pull in an pike-based version of magnum with this change applied.
-    RUN pip install -U --no-deps git+https://github.com/stackhpc/magnum.git@stackhpc/pike
+    # This branch fixes k8s FailedNodeAllocatableEnforcement warning
+    RUN pip install -U --no-deps git+https://github.com/stackhpc/magnum@stackhpc/queens
+  sahara_api_footer: |
+    # Bring in hack to the vanilla plugin
+    RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/pike
+  sahara_engine_footer: |
+    # Bring in hack to the vanilla plugin
+    RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/pike
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml]: Where have these come from?

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/kayobe/kolla.yml [hunk 3]
```diff
@@ -33,11 +33,11 @@
 #kolla_ansible_source_path:
 
 # URL of Kolla Ansible source code repository if type is 'source'.
-#kolla_ansible_source_url:
+kolla_ansible_source_url: https://www.github.com/stackhpc/kolla-ansible
 
 # Version (branch, tag, etc.) of Kolla Ansible source code repository if type
 # is 'source'.
-#kolla_ansible_source_version:
+kolla_ansible_source_version: stackhpc/pike
```
Comment (dougszumski) [inline etc/kayobe/kolla.yml:40]: This is also the default: ```kolla_ansible_source_version: "stackhpc/pike"```

**Review by dougszumski — CHANGES_REQUESTED** [review]

**Review by markgoddard — APPROVED** [review]

**Comment by markgoddard** [comment]
Just merged the kolla.yml defaults into alt-1, so let's stick with them here.

**Pull request merged**